test/chart/LineChart.spec.tsx

```diff
@@ -857,11 +857,13 @@ describe('<LineChart />', () => {
   });
 
   describe('LineChart - test ref access', () => {
-    test('should allow access to the CategoricalChartWrapper through the ref prop forwarded from CategoricalChart', () => {
-      // TODO we need to fix the ref typing in generateCategoricalChart, this class is created dynamically and cannot be typed
-      // @ts-expect-error TODO remove in followup PR
-      // eslint-disable-next-line no-undef
-      let refNode: CategoricalChartWrapper | null = null;
+    test('should allow access to the main SVG through the ref prop forwarded from CategoricalChart', () => {
+      /*
+       * This is a breaking change from v2.0.0, where the ref was not forwarded
```

Comment:
@ckifer where should we point the `ref`? It can't be the class instance anymore (because there is no class to point to). Should it reference the div wrapper, or the SVG?